chaincode/testtuple.go

```diff
@@ -264,21 +266,35 @@ func logStartTest(db *LedgerDB, args []string) (outputTesttuple outputTesttuple,
 	if err != nil {
 		return
 	}
+
+	tuple, err := db.GetGenericTuple(testtuple.TraintupleKey)
+	if err != nil {
+		return outputTesttuple{}, err
+	}
+
+	// cancel testtuple if compute plan is canceled
+	if tuple.ComputePlanID != "" {
+		err := cancelIfComputePlanIsCanceled(db, inp.Key, tuple.ComputePlanID, &testtuple)
```

Comment:
I think there is a problem here: we should return after canceling this tuple. If we continue, we will update the status later in the function to `StatusDoing` or am I missing something obvious?

Reply:
yes, I will fix that

Reply:
fixed here 681506d